src/Stack/Options.hs

```diff
@@ -682,7 +683,12 @@ initOptsParser :: Parser InitOpts
 initOptsParser =
     InitOpts <$> solver <*> omitPackages
              <*> overwrite <*> fmap not ignoreSubDirs
+             <*> searchDirs
   where
+    searchDirs =
+      many (textArgument
+              (metavar "SEARCH-DIRECTORIES" <>
+               help "Directories which are searched for cabal files. If non specified, uses current directory."))
```

Comment:
How about a more concise help like `Directories to include, default is current directory.`?